go/libkb/env.go

```diff
@@ -221,33 +221,12 @@ func (e *Env) GetUpdaterConfig() UpdaterConfigReader {
 }
 
 func (e *Env) GetMountDir() (string, error) {
-	var darwinMountsubdir string
-	runMode := e.GetRunMode()
-	switch runMode {
-	case DevelRunMode:
-		darwinMountsubdir = "keybase.devel"
-	case StagingRunMode:
-		darwinMountsubdir = "keybase.staging"
-	case ProductionRunMode:
-		darwinMountsubdir = "keybase"
-	default:
-		return "", fmt.Errorf("Invalid run mode: %s", runMode)
-	}
-
 	return e.GetString(
 		func() string { return e.cmd.GetMountDir() },
 		func() string { return os.Getenv("KEYBASE_MOUNTDIR") },
 		func() string { return e.GetConfig().GetMountDir() },
 		func() string {
-			switch runtime.GOOS {
-			case "darwin":
-				return filepath.Join(
-					string(filepath.Separator), darwinMountsubdir)
-			case "linux":
-				return filepath.Join(e.GetDataDir(), "fs")
-			default:
-				return ""
-			}
+			return filepath.Join(e.GetDataDir(), "fs")
```

Comment:
I'm not sure what's best, I thought this would best communicate "Keybase FS".  But in any case, the Linux builds are already live with "fs" and changing it would be a huge pain in the butt now.  Sorry, it might be a good suggestion, but I think it's one PR too late.

Reply:
Ah sorry I didn't think of that earlier. I'm OK with `fs` too!